Sources/PostgreSQL/Database/PostgreSQLDatabaseConfig.swift

```diff
@@ -21,18 +21,22 @@ public struct PostgreSQLDatabaseConfig {
     
     /// Optional password to use for authentication.
     public let password: String?
+
+    /// Optional ssl connection
+    public let shouldPreferSSL: Bool?
     
     /// Creates a new `PostgreSQLDatabaseConfig`.
-    public init(hostname: String, port: Int = 5432, username: String, database: String? = nil, password: String? = nil) {
+    public init(hostname: String, port: Int = 5432, username: String, database: String? = nil, password: String? = nil, shouldPreferSSL: Bool? = false) {
```

Comment:
How about making this just a `Bool` instead of a `Bool?`? (Also below and above.)

Reply:
I used an optional here to keep backwards compatibility

Reply:
You don't need an optional for that, a regular Bool with a default value is also backwards compatible.